Adjust the look of the analyse result page

Blackgen opened the pull request:
-  Add padding zeros to silver/copper if the value is >10
  -  Add Item-Type
  -  Add Table-header

Blackgen committed:
Adjust the look of the analyse result page

  -  Add padding zeros to silver/copper if the value is >10
  -  Add Item-Type
  -  Add Table-header

immortius commented on site/js/main.js:
```diff
@@ -174,15 +174,20 @@ function updateThresholdCalculation() {
-			  + ' <span id="min-unlock-copper">' + copper + '</span><span class="base-icon copper-icon" role="img" aria-label="Copper"></span></div></div>';
-	}	
-	
+			  + '<div class="entry-name">' + item.name +'</div><div class="entry-name">' + capitalizeFirstLetter(item.section) + '</div><div class="entry-price"><span>' + gold + '</span><span class="base-icon gold-icon" role="img" aria-label="Gold"></span>'
```
I think this is the one thing that gives me pause. ```item.section``` is technically the id of the section, where a display name for the section is captured in metadata - although not as a mapping. Theoretically we could pull out that mapping as a lookup and get the display from there. This would mean that, for example, the label for a back piece would be "Back Items" rather than "Back"... which still isn't great because of the plural I guess. Thoughts?

Blackgen replied:
Yea I knew that I'm misusing the section property for this..
However in my testing with an almost empty account this looks fine, only the Back-Items are the only exception beeing listed with only using the term "Back"

I was mostly interessted in seeing this for dyes as the pure name doen't always imply that it is a dye.

I think adding the singular name to the config would be best. The pluar form just won't sound right for a single item.
I'd add this as a field called "type" or "name_singular" to each section.

immortius replied:
I like the idea of adding an extra field for it, provided through the config. :)

Blackgen committed:
Add trading post name to skins with differing name

immortius committed:
* Switched out Unstable Cosmic Essence for Unstable Fractal Essence
 * Updates for new skins as of halloween 2020
 * Added support for Fractal Journal as currency
 * Added Icebrood Saga currency filter.

immortius committed:
Updated for Halloween/Wintersday/DRMs.

immortius committed:
Updated for new items.
Corrected handling of "silver" and "gold" currencies from wiki.
Added exotic and legendary essence of luck currencies.

immortius committed:
Updates up to the 29th of April 2021

immortius committed:
Added light and dark themes.

Blackgen commented:
Hm the rebase did not turn out as I expected.
Please ignore this PR until I'm done, I'll let you know once this is ready for a review

Blackgen commented:
Sorry, I don't have the time to fix this.

If you want you can cherry pick and adjust the changes.
